src/projects/spades/pipeline/spades_pipeline/executors/executors.py

```diff
@@ -30,5 +31,217 @@ def dump_commands(self, commands, outputfile):
         pass
 
     @abstractmethod
-    def touch_file(self, command):
+    def join(self, job_name):
         pass
+
+    @abstractmethod
+    def kill(self, job_name):
+        pass
+
+
+class ExecutorCluster(ExecutorBase):
+    grid_engine = None
```

Comment:
isn't it supposed to be inside `__init__()` ? 
this defines static members, is it on purpose?